CHANGES.md

```diff
@@ -62,18 +54,28 @@ replace with next unreleased version
 - Open bugs affecting this version
 	- [Jira view of open Issues](https://codice.atlassian.net/issues/?jql=project%3D%20%22Codice%20Alliance%22%20AND%20issuetype%20%3D%20bug%20%20AND%20status%20%3D%20Open%20AND%20affectedVersion%20%3D%200.2.0%20ORDER%20BY%20priority) affecting this version.
 
-## 0.1.3
-	Release Build Date: pending
+
+## 0.2.0
+	Release Date: 2016-2-8
+
+### NOTES
+
+- Built on [DDF v.2.10.0](https://github.com/codice/ddf/blob/master/CHANGES.md#2100)
+
+### NEW FEATURES
+
+- new features
+	- [JIRA View of All New Features](https://codice.atlassian.net/issues/?jql=project%20%3D%20%22Codice%20Alliance%22%20and%20issuetype%20in%20%28story%2Ctask%29%20AND%20resolution%20%3D%20done%20AND%20fixVersion%20%3D%200.2.0%20ORDER%20BY%20summary%20ASC%2C%20priority)
 
 ### ISSUES RESOLVED
 
-- Bug fixes
-	- [Jira view of all issues resolved](https://codice.atlassian.net/issues/?jql=project%3D%22Codice%20Alliance%22%20and%20issuetype%20%3D%20bug%20AND%20fixVersion%20%3D%200.1.3%20and%20resolution%20%3D%20done) in this version.
+- Important issues resolved
+	- [Jira view of all issues resolved](https://codice.atlassian.net/issues/?jql=project%3D%22Codice%20Alliance%22%20AND%20type%20%3D%20Bug%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20%3D%200.2.0%20ORDER%20BY%20priority) in this version.
```

Comment:
Currently, the CAL Kanban board does not have a Fixed status, so this would always be empty